src/procedures/proc_fulltext_query.c

```
@@ -170,3 +170,170 @@
+
+ProcedureResult Proc_FulltextQueryRelationshipFree
+(
+	ProcedureCtx *ctx
+) {
+	// Clean up.
+	if(!ctx->privateData) return PROCEDURE_OK;
+
+	QueryRelationshipContext *pdata = ctx->privateData;
+	array_free(pdata->output);
+	if(pdata->iter) RediSearch_ResultsIteratorFree(pdata->iter);
+	rm_free(pdata);
```

cppcheck (2.10-2)

[error] Common realloc mistake

(memleakOnRealloc)